crates/turbopack-ecmascript-runtime/js/src/shared-node/node-externals-utils.ts

```diff
@@ -1,5 +1,21 @@
 /// <reference path="../shared/runtime-utils.ts" />
 
+declare var RUNTIME_PUBLIC_PATH: string;
+declare var OUTPUT_ROOT: string;
+declare var ASSET_PREFIX: string;
+
+const path = require("path");
```

Comment:
This probably breaks on `edge`. edge also used `node-externals-utils.ts`

Reply:
You probably need to move that into a separate file

Reply:
Updated, and passing next.js CI : https://github.com/vercel/next.js/pull/58987